lib/eex/lib/eex/tokenizer.ex

```diff
@@ -73,13 +74,11 @@ defmodule EEx.Tokenizer do
 
   # Retrieve marker for <%
 
-  defp retrieve_marker('=' ++ t) do
-    {'=', t}
-  end
+  defp retrieve_marker([marker | t]) when marker in [?=, ?/, ?|],
+    do: {[marker], t}
 
```

Comment:
Please don't leave a blank line between function clauses when using the `do:` notation. :)